transformer_engine/pytorch/attention.py

```diff
@@ -41,6 +42,19 @@
 __all__ = ["DotProductAttention"]
 
 
+@jit_fuser
+def get_cu_seqlens(padding_mask: torch.Tensor) -> torch.Tensor:
+    """
+    Given a padding mask of shape [seq_len, batch_size], returns an int32
```

Comment:
I guess by `seq_len` you mean `max_seqlen`? Also, should the mask tensor be required to be a CUDA tensor?

Reply:
Yeah it is the `max_seqlen`.

Reply:
Yeah it should be cuda tensor since the inp is cuda as well.